buildSrc/src/main/groovy/accord.java-conventions.gradle

```diff
@@ -49,10 +49,6 @@ test {
     jvmArgs += ['-XX:+HeapDumpOnOutOfMemoryError', "-XX:HeapDumpPath=${buildDir}"]
 }
 
-tasks.withType(Test) {
```

Comment:
looks like rebase issue that got merged... I added this, Benedict rebased onto my branch and added similar logic (see line 49), but didn't detect the logical conflict so this got merged.